### PR TITLE
Feature/mercadopago v2

### PR DESCRIPTION
**Pull request by NicholasPedroso**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/online-payments/checkout-api/previous-requirements.es.md [hunk 1]
```diff
@@ -24,6 +24,21 @@ Es importante que siempre utilices nuestras librerías oficiales para tu integra
 >
 > &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incluye MercadoPago.js
 
+----[mlb]----
+> INFO
+>
+> Nueva versión MercadoPago.js
+>
+> Utiliza la librería MercadoPago.js V2 para actualizarte a la última versión, autogenerar la lógica de negocio en tu formulario de pagos con tarjeta y mejorar la compatibilidad con los distintos navegadores.<br><br>[Integrar Checkout API con MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-api/previous-requirements-v2).
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/previous-requirements.es.md:32]:
```suggestion
> Utiliza la librería MercadoPago.js V2 para actualizarte a la última versión, autogenerar la lógica de negocio en tu formulario de pagos con tarjeta y mejorar la compatibilidad con los distintos navegadores.<br><br>[Integrar Checkout Transparente con MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-api/previous-requirements-v2)
```

File: guides/online-payments/checkout-api/previous-requirements.es.md [hunk 2]
```diff
@@ -24,6 +24,21 @@ Es importante que siempre utilices nuestras librerías oficiales para tu integra
...
+------------
+----[mla, mlm, mco, mlc, mlu]----
+> INFO
+>
+> Nueva versión MercadoPago.js
+>
+> Utiliza la librería MercadoPago.js V2 para actualizarte a la última versión, autogenerar la lógica de negocio en tu formulario de pagos con tarjeta y mejorar la compatibilidad con los distintos navegadores.<br><br>[Integrar Checkout Transparente con MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-api/previous-requirements-v2).
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/previous-requirements.es.md:39]:
```suggestion
> Utiliza la librería MercadoPago.js V2 para actualizarte a la última versión, autogenerar la lógica de negocio en tu formulario de pagos con tarjeta y mejorar la compatibilidad con los distintos navegadores.<br><br>[Integrar Checkout API con MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-api/previous-requirements-v2)
```

File: guides/online-payments/checkout-api/receiving-payment-by-card.pt.md [hunk 3]
```diff
@@ -42,6 +42,22 @@ Tanto para o frontend como para o backend, recomendamos utilizar [nossos SDKs](h
...
+> Nova versão de MercadoPago.js
+>
+> Utilize a biblioteca MercadoPago.js V2 para criar seu formulário de cartão com a funcionalidade CardForm e autogerar a lógica de negócio necessária para fazer o pagamento.<br><br>[Integrar Checkout Transparente com MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-api/receiving-payment-by-card-v2).
+------------
+----[mla, mlm, mpe, mco, mlu, mlc]----
+> INFO
+>
+> Nova versão de MercadoPago.js
+>
+> Utilize a biblioteca MercadoPago.js V2 para criar seu formulário de cartão com a funcionalidade CardForm e autogerar a lógica de negócio necessária para fazer o pagamento.<br><br>[Integrar Checkout API com MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-api/receiving-payment-by-card-v2).
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/receiving-payment-by-card.pt.md:57]:
```suggestion
> Utilize a biblioteca MercadoPago.js V2 para criar seu formulário de cartão com a funcionalidade CardForm e autogerar a lógica de negócio necessária para fazer o pagamento.<br><br>[Integrar Checkout API com MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-api/receiving-payment-by-card-v2)
```

File: guides/online-payments/checkout-api/receiving-payment-by-card.pt.md [hunk 4]
```diff
@@ -42,6 +42,22 @@ Tanto para o frontend como para o backend, recomendamos utilizar [nossos SDKs](h
 >
 > Capture os dados de cartão
 
+----[mlb]----
+> INFO
+>
+> Nova versão de MercadoPago.js
+>
+> Utilize a biblioteca MercadoPago.js V2 para criar seu formulário de cartão com a funcionalidade CardForm e autogerar a lógica de negócio necessária para fazer o pagamento.<br><br>[Integrar Checkout Transparente com MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-api/receiving-payment-by-card-v2).
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/receiving-payment-by-card.pt.md:50]:
```suggestion
> Utilize a biblioteca MercadoPago.js V2 para criar seu formulário de cartão com a funcionalidade CardForm e autogerar a lógica de negócio necessária para fazer o pagamento.<br><br>[Integrar Checkout Transparente com MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-api/receiving-payment-by-card-v2)
```

File: guides/online-payments/checkout-api/receiving-payment-by-card.es.md [hunk 5]
```diff
@@ -42,6 +42,21 @@ Tanto para el frontend como para el backend, recomendamos utilizar [nuestras lib
 >
 > Captura los datos de la tarjeta
 
+----[mlb]----
+> INFO
+>
+> Nueva versión MercadoPago.js
+>
+> Utiliza la librería MercadoPago.js V2 para crear tu formulario de tarjeta con la funcionalidad CardForm y autogenerar la lógica de negocio necesaria para realizar el pago.<br><br>[Integrar Checkout Transparente con MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-api/receiving-payment-by-card-v2).
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/receiving-payment-by-card.es.md:50]:
```suggestion
> Utiliza la librería MercadoPago.js V2 para crear tu formulario de tarjeta con la funcionalidad CardForm y autogenerar la lógica de negocio necesaria para realizar el pago.<br><br>[Integrar Checkout Transparente con MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-api/receiving-payment-by-card-v2)
```

File: guides/online-payments/checkout-api/receiving-payment-by-card.es.md [hunk 6]
```diff
@@ -42,6 +42,21 @@ Tanto para el frontend como para el backend, recomendamos utilizar [nuestras lib
...
+------------
+----[mla, mlm, mpe, mco, mlu, mlc]----
+> INFO
+>
+> Nueva versión MercadoPago.js
+>
+> Utiliza la librería MercadoPago.js V2 para crear tu formulario de tarjeta con la funcionalidad CardForm y autogenerar la lógica de negocio necesaria para realizar el pago.<br><br>[Integrar Checkout API con MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-api/receiving-payment-by-card-v2).
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/receiving-payment-by-card.es.md:57]:
```suggestion
> Utiliza la librería MercadoPago.js V2 para crear tu formulario de tarjeta con la funcionalidad CardForm y autogenerar la lógica de negocio necesaria para realizar el pago.<br><br>[Integrar Checkout API con MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-api/receiving-payment-by-card-v2)
```

File: guides/online-payments/checkout-api/receiving-payment-by-card.en.md [hunk 7]
```diff
@@ -29,6 +29,12 @@ Both for frontend and backend, we recommend [our libraries](https://www.mercadop
 >
 > Card data capture
 
+> INFO
+>
+> New MercadoPago.js version
+>
+> Use MercadoPago.js V2 library to create your card form with CardForm functionality, which includes all business logic needed to make the payment.<br><br>[Integrate Checkout API with MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-api/receiving-payment-by-card-v2).
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/receiving-payment-by-card.en.md:36]:
```suggestion
> Use MercadoPago.js V2 library to create your card form with CardForm functionality, which includes all business logic needed to make the payment.<br><br>[Integrate Checkout API with MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-api/receiving-payment-by-card-v2)
```

File: guides/online-payments/checkout-api/other-payment-ways.en.md [hunk 8]
```diff
@@ -126,6 +126,12 @@ Keep in mind that the answer will return all the payments methods. For this reas
 >
 > Data capture for payment
 
+> INFO
+>
+> New MercadoPago.js version
+>
+> Use MercadoPago.js V2 library to be updated to the latest version and to improve compatibility with different browsers.<br><br>[Integrate Checkout API with MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-api/other-payment-ways).
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/other-payment-ways.en.md:133]:
```suggestion
> Use MercadoPago.js V2 library to be updated to the latest version and to improve compatibility with different browsers.<br><br>[Integrate Checkout API with MercadoPago.js V2](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-api/other-payment-ways)
```

File: guides/online-payments/checkout-api/receiving-payment-by-card-core-methods-v2.es.md [hunk 9]
```diff
@@ -0,0 +1,600 @@
...
+------------
+----[mla, mlm, mpe, mco, mlu, mlc]----
+La integración del Checkout API de Mercado Pago para tarjetas permite que puedas ofrecer una opción de pagos completa dentro de tu sitio. Toda la experiencia sucede en tu tienda para que los clientes no tengan que salir al momento de realizar la compra.
+------------
+
+Usa los [ejemplos descargables](#bookmark_ejemplos_descargables) para conocer la integración completa o para adaptarlos según lo que necesites.
+
+## ¿Cómo funciona?
+
+![API-integration-flowchart](/images/api/api-integration-flowchart-coremethods-es.png)
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/receiving-payment-by-card-core-methods-v2.es.md:19]: Corregir contenidos en portugués en la imagen.

File: guides/online-payments/checkout-api/receiving-payment-by-card-v2.es.md [hunk 10]
```diff
@@ -0,0 +1,515 @@
...
+------------
+----[mla, mlm, mpe, mco, mlu, mlc]----
+La integración del Checkout API de Mercado Pago para tarjetas permite que puedas ofrecer una opción de pagos completa dentro de tu sitio. Toda la experiencia sucede en tu tienda para que los clientes no tengan que salir al momento de realizar la compra.
+------------
+
+Usa los [ejemplos descargables](#bookmark_ejemplos_descargables) para conocer la integración completa o para adaptarlos según lo que necesites.
+
+## ¿Cómo funciona?
+
+![API-integration-flowchart](/images/api/api-integration-flowchart-cardform-es.png)
```
Comment (pamalv7) [inline guides/online-payments/checkout-api/receiving-payment-by-card-v2.es.md:19]: Corregir contenido de imagen. 
"Formulario" va sin tilde.

**Review by pamalv7 — CHANGES_REQUESTED** [review]

**Review by pamalv7 — APPROVED** [review]

**Review by nicolassalvanes — APPROVED** [review]

**Pull request merged**